db/migrate/20220613205853_add_confirmable_to_devise_v1.rb

```diff
@@ -0,0 +1,8 @@
+class AddConfirmableToDeviseV1 < ActiveRecord::Migration[7.0]
+  def change
+    change_table(:users) do |t|
+       t.confirmable 
```

Comment:
![image](https://user-images.githubusercontent.com/65068771/173456791-89c2deaa-e786-4f08-aba6-ea1a9aec0a1c.png)

- [ ] I am getting this error when running your migrations. this is because there is no data type in Postgres called  `confirmable`. to fix this kindly delete this migration because the confirmable can be enabled in the main devise migration :point_down: you just need to uncomment it.

db/migrate/20220613193104_add_devise_to_users.rb
![image](https://user-images.githubusercontent.com/65068771/173457948-d3220da5-a1d9-46ca-bf68-bc0747e52d6e.png)